README.md

```diff
@@ -19,50 +17,67 @@ A few Sphinx extensions are used as well:
   in the API docs and tutorials. This is also modified from its original state, and that
   fork can be found at [this fork](https://github.com/purduesigbots/sphinx-tabs).
 
-## Contributing
+## I Just Want to Help
+Great! Please help us. Choose one of the following that best suits your needs. 
+
+#### Easy
+If you want ot change a page or a couple words, it is simple. All you have to do is to edit a file and [Github's UI](https://help.github.com/en/articles/editing-files-in-another-users-repository) will take care of the rest of it! Then you have officially contributed to `pros-docs`! 
+
+#### Medium
+Here you will install a basic html version of `pros-docs` This is adequate for most people
+To make this process easier. If you don't already have a basic understanding of how these work:       [git](https://rogerdudler.github.io/git-guide/), 
+                  [github](https://github.com/firstcontributions/first-contributions), 
+                  [sphinx](https://www.mageworx.com/blog/2016/05/sphinx-the-beginners-guide/), 
+                  [python3](https://wiki.python.org/moin/BeginnersGuide), you should go check them out before you start.
+ 
+ Clone the project, change the directory to `pros-docs`
+ ```sh
+git clone https://github.com/purduesigbots/pros-docs

```

Comment:
you can make the next steps regarding submodules by running `git clone --recursive ...`. if someone forgets to add the flag when cloning, they can `git submodule update --init --recursive` to do all the rest in one step (and `git submodule update --recursive` to update later)

Reply:
I added a note below that code. Let me know if that solves it.